### ISSUE TITLE
breeze: context menu looks/feels like "old xfce"

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.9#235]
i am no designer, so i dont know what it makes it that way, but it certainly doesnt look/give a modern feeling, quite the opposite.
Maybe spacing, text and general alignment, could be compared to modern Windows10 or MacOS or Chrome os:
![Screenshot_20200723_191917](https://user-images.githubusercontent.com/840070/88317576-c15a0480-cd19-11ea-8bc1-13a078ec75e8.png)

Above is breez dark, but breeze white is equally old look.

**Comment by star-buck** [comment]
look at that spacing:
![Screenshot_20200723_192238](https://user-images.githubusercontent.com/840070/88317706-f36b6680-cd19-11ea-9ea6-9927a564d4f4.png)
beginning of X...

**Comment by star-buck** [comment]
win10:
![win10](https://user-images.githubusercontent.com/840070/88347127-4e1cb680-cd4a-11ea-9cb0-10f854e35198.jpg)

**Comment by Sodivad** [comment]
Windows 10 is the same concept as ours with a bit more spacing and always reserving some space to the left even when the submenu has no icons it looks like

**Comment by zzag** [comment]
> always reserving some space to the left even when the submenu has no icons it looks like

For what it's worth, I made a patch to do the same thing in Breeze a couple of years ago. To me, the lack of space on left side feels awkward and a bit unnatural, I don't know why, it just feels that way... Anyway, that patch was negatively received.

**Comment by star-buck** [comment]
I think its about the spacing too and recall numerous comments that KDE
"doesnt get it" right

**Comment by Manueljlin** [comment]
![Context Menu](https://user-images.githubusercontent.com/11275573/88390748-2add2000-cdb9-11ea-8e2c-d91f86f9de10.png)

**Comment by star-buck** [comment]
Please suggest upstream ticket, the currebt looks absolutely crammed if you
just look at top and bottom entries...

**Comment by Pointedstick** [comment]
CCing @noahadvs as well, as he actually has a WIP patch to fix this I think!

**Comment by Manueljlin** [comment]
Yes, he has a patch for the new highlight style (looks great :D)
I'll post the padding measurements I used soon

**Comment by Manueljlin** [comment]
I'm back! I was afk for a while, sorry. Here's the measurements.

![Context Menu](https://user-images.githubusercontent.com/11275573/88427667-abb90d80-cdf3-11ea-914f-ff62daad163a.png)

While here I'm using 10px of padding at the left side, it can use 12px just fine if that's easier and cleaner to code (same goes for the vertical 7 pixels, I just wanted to make the size match the buttons size but if that doesn't matter we can use 8px just fine). The click/touch area is the same as most tabs and buttons (30px) which is probably the bare minimum at making it touch friendly.

I've also used the view color instead of the mix of view background and application background to make the hierarchy a bit clearer.
 Will make a phab task soon.

**Comment by Manueljlin** [comment]
> Please suggest upstream ticket, the current looks absolutely crammed if you just look at top and bottom entries...

done @ https://phabricator.kde.org/T13442
